### PR TITLE
CI: split workflow files

### PR DESCRIPTION
**Pull request by viktormalik**
So far, we put all CI actions into a single workflow file which is getting quite huge. This separates some jobs into their own workflow files.

### CODE REVIEW

File: .github/workflows/viewer.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    - name: Run Cypress tests
+      uses: cypress-io/github-action@v6
+      with:
+        install: false
+        component: true
+        browser: firefox
+        working-directory: view
+        config: video=false,screenshotOnRunFailure=false
+    - name: Check coding style with ESLint
+      run: npm run lint --prefix view
```
Comment (PLukas2018) [inline .github/workflows/viewer.yml:33]: I am wondering if we do not want to move this to the `code-style.yml` file?

**Review by PLukas2018 — APPROVED** [review]
Looks good to me. I have just one note.

File: .github/workflows/viewer.yml (hunk 1 above)
Comment (viktormalik) [reply .github/workflows/viewer.yml:33]: Yeah, good idea. Does it make sense to run the linter for each Node version? Or just the latest one is fine?

**Comment by viktormalik** [comment]
I did some changes to the code-style workflow (split into multiple jobs), another review would be useful.

File: .github/workflows/viewer.yml (hunk 1 above)
Comment (PLukas2018) [reply .github/workflows/viewer.yml]: I think that just the latest one should be enough.

**Review by FrNecas — APPROVED** [review]

**Pull request merged**